### ISSUE TITLE
submit key does not work in sensingkit  driver

### ISSUE THREAD

**Issue by tsh-box** [me-box/databox#147]
hello,

I installed sensingkit driver and I inserted the IP address of Databox mobile app and then I clicked the submit but it seems there is no reaction .

![tocon1](https://user-images.githubusercontent.com/29420329/31533795-02dc81e0-b02f-11e7-934b-56d99d49fe56.png)

**Comment by Toshbrown** [comment]
This should work now but you will need the new app from the app store links are on the driver config page

**Comment by tsh-box** [comment]
Thanks you,
But there is no reaction when I clicked the Google Play button .

![tocon3](https://user-images.githubusercontent.com/29420329/31591889-029fd8da-b25f-11e7-9fde-daa7447d465a.png)

**Comment by Toshbrown** [comment]
You can search in the app store for Databox (its made by Horizon DER Institute) 

Its here https://play.google.com/store/apps/details?id=io.databox.app

@ktg can you check the link please?

**Comment by tsh-box** [comment]
I have already installed in window10.But I can't use.
It can only install in android,right?

**Comment by tsh-box** [comment]
I installed the Databox app in the android os.
And databox conatainer UI are also run in dev mode on window10.
After installed Databox app in android os ,it asked the Databox URL and then I added the  databox conatainer UI address,it connected.
And then I selected  the Mobile sensor data and installed the driver-sensingkit in android os.

![ccc2](https://user-images.githubusercontent.com/29420329/31604475-64fbcbd0-b29e-11e7-9309-0f50b391a1aa.png)

And then,I inserted the databox mobile app address and clicked submit button,but it was also no reaction.

![ccc1](https://user-images.githubusercontent.com/29420329/31604299-d4db4c92-b29d-11e7-8ccd-969c8cda2e85.png)
